sensors/nxt_i2c_sensor.h

```diff
@@ -158,6 +159,7 @@ enum nxt_i2c_sensor_type {
  * termination, so no more than 18 chars here!
  */
 #define LEGO_NXT_ULTRASONIC_SENSOR_NAME		"lego-nxt-us"
+#define LEGO_NXT_TEMPERATURE_SENSOR_NAME		"lego-nxt-temp"
```

Comment:
In <https://github.com/kortschak/lego-linux-drivers/blob/fb80a2581826838a11c9d3773a201298c48112ac/sensors/nxt_i2c_sensor.h>, it looks like there are too many tabs here.